test/tests/transfers/multiapp_nearest_node_transfer/tests

```diff
@@ -3,49 +3,73 @@
     type = 'Exodiff'
     input = 'tosub_master.i'
     exodiff = 'tosub_master_out_sub0.e'
+    design = 'transfers/MultiAppNearestNodeTransfer.md'
+    issues = '#1866'
+    requirement = "The NearestNodeTransfer shall transfer both nodal and elemental fields to a SubApp from the Master App."
```

Comment:
Design inside requirement. Should say "The system shall...". `NearestNodeTransfer` is a product of design.

Also below.

Reply:
Fixed